app/components/Contacts/ContactsPanel/ContactsPanel.scss

```diff
@@ -102,6 +103,12 @@
   transform: translate(-50%, -50%);
 }
 
+.groupHeader {
+  padding: 10px 24px;
+  background-color: #F6F7F8;
```

Comment:
Just curious where you got this color from in the mocks its `#979797` which is also not on the palette 😆... Either way we should probably be referencing one of the color vars here so that we can swap out if needed for alternate themes in the future

Reply:
As discussed, we will consolidate this when we get to the dark theme.